src/util/game-activity-tracker/admin.ts

```diff
@@ -57,9 +58,12 @@ export async function adminWhitelistgame(interaction: ChatInputCommandInteractio
 	let blacklistedGames: string[] | undefined = activityTrackerBlacklistDb.get("general-game");
```

Comment:
We could avoid this entire erorr handler below, by switching get to ensure

Reply:
@WilsontheWolf don't really know enmap
if its important to you just add it and feel free to merge after that :)